precompiles/utils/tests/solidity_test.sol

```diff
@@ -1,39 +1,25 @@
 // SPDX-License-Identifier: GPL-3.0-only
 pragma solidity >=0.8.0;
 
-interface Randomness {
+interface SolidityTest {
     /// Function without params
     function fnNoArgs() external;
 
     /// @dev Interface for all randomness consumers
     ///
     /// @param refund_address Address to refund with fee less cost of subcall
     /// Selector: c4921133
-    function fnOneArg(
-        address refund_address,
-    ) external;
+    function fnOneArg(address refund_address) external;
```

Comment:
```suggestion
    function fnOneArg(address a) external;
```
This is not for randomness anymore so all inputs should be renamed something more generic